style.css

```diff
@@ -95,3 +95,48 @@ button:focus-visible {
     background-color: #f9f9f9;
   }
 }
+
+
+.tooltip-container {
+    position: relative;
+}
+
+.tooltip {
```

Comment:
You have missed an arrow for tooltip. You can add an arrow with :before and :after pseudo-elements to avoid extra markdown.